docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-installer-iso.md

```diff
@@ -136,9 +136,31 @@ Use the following instructions to build the Edge Installer ISO. The optional ste
    through Local UI. This includes both the content bundle and cluster definition. For more information, refer to
    [Embed Public Key in Edge Artifacts](./signed-content.md).
 
+### Provide Client Certificates for Proxy Servers
+
+8. (Optional) This step is only required if your builds occur in a proxied network environment, and your proxy servers
```

Comment:
this is not accurate. If someone has their base image in a registry needs certs, they also need to place registry cert into certs dir. But in this case, its not behind proxy